Mathlib/SetTheory/Ordinal/Arithmetic.lean

```diff
@@ -768,7 +768,7 @@ instance div : Div Ordinal :=
 theorem div_zero (a : Ordinal) : a / 0 = 0 :=
   dif_pos rfl
 
-theorem div_def (a) {b : Ordinal} (h : b ≠ 0) : a / b = sInf { o | a < b * succ o } :=
+private theorem div_def (a) {b : Ordinal} (h : b ≠ 0) : a / b = sInf { o | a < b * succ o } :=
```

Comment:
Are you sure this one should be private? This looks like a useful description of division

Reply:
What we really need is `a / b = o` iff `o * b ≤ a < o * b + o`, which we basically already have.